test/test_sox_compatibility.py

```diff
@@ -78,16 +78,16 @@ def test_lowpass(self):
         Test biquad lowpass filter, compare to SoX implementation
         """
 
-        CUTOFF_FREQ = 3000
```

Comment:
Thanks for fixing all the format. But could you open a separate PR from separate branch, so that we can focus `dcshit` in this PR?

Reply:
Removed those from this PR . Will submit them in another PR